tasks/main.yml

```diff
@@ -11,10 +12,10 @@
       supported OS"
   when: (release is not defined) or (release_url[release] is not defined)
 
-- name: Include OS-specific tasks
-  ansible.builtin.include_tasks: "{{ ansible_os_family | lower }}.yml"
+- name: Configure Repository
+  ansible.builtin.include_tasks: "{{ ansible_os_family | lower}}.yml"
+  # when: (ansible_distribution == "RedHat") or (ansible_distribution == "CentOS") or (ansible_distribution == "Scientific") or (ansible_distribution == "AlmaLinux")
```

Comment:
I tend to prefer removing unused code, than keeping commented stuff in the main branch.